src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -1566,6 +1566,8 @@ int32_t SurgeGUIEditor::controlModifierClicked(CControl* control, CButtonState b
                spawn_miniedit_text(synth->storage.getPatch().CustomControllerLabel[ccid], 16);
                ((CModulationSourceButton*)control)
                    ->setlabel(synth->storage.getPatch().CustomControllerLabel[ccid]);
+                control->setDirty();
+                control->invalid();
                //((gui_slider*)metaparam[ccid])->setLabel(synth->storage.getPatch().CustomControllerLabel[ccid]);
```

Comment:
Can we remove the commented out code or is desperately needed?

Reply:
Not really any type of review comment for this. Just wondering.

Reply:
So since the code is kinda historied and we are learning about it I’ve decided to not remove commented code in case it holds some nugget of past wisdom we need in the future. Thanks for the reviews on these issues!

Reply:
Makes sense, agreed.